.gitignore

```diff
@@ -2,6 +2,7 @@
 config.yaml
 ssgbot.env
 docker-compose.yaml
+classify.jpg
```

Comment:
Either this or see if we can use `BytesIO` for this.

```suggestion
classify.jpg
tts.ogg
```

Reply:
not needed as of [this commit](https://github.com/Super-Serious/bot/commit/b36bd2f027e14cfddda040bef599b653ca91f700)